.github/workflows/consumer_contract_tests.yml

```diff
@@ -52,6 +52,7 @@ env:
 jobs:
   bump-check:
     runs-on: ubuntu-latest
+    if: false # Prevent running since we removed BPM which was the only provider for Rawls.
```

Comment:
just delete this whole workflow instead of setting to false? Since the only contract is Rawls->BPM, there's no point in running pact tests at all. What do you think?